### ISSUE TITLE
Deconvolution file corresponding to 1160920F Patient counts from the zenodo repository.

### ISSUE THREAD

**Issue by ZheFrench** [Swarbricklab-code/BrCa_cell_atlas#20]
For the patient 1160920F  related to here https://zenodo.org/record/4739739#.Y007vNJBxhE, I would like to know which deconvolution ouput is corresponding in here : 
https://github.com/Swarbricklab-code/BrCa_cell_atlas/tree/main/spatial_analysis/stsc/major
[182963_1160920F-count-matrix.tsv]
[182968_1160920F-count-matrix.tsv]
[182969_1160920F-count-matrix.tsv]

From the metadata, here.
https://github.com/Swarbricklab-code/BrCa_cell_atlas/blob/main/spatial_analysis/Meta_Data.csv
I bet it's 182968_1160920F. 
Right ? Thanks.

**Comment by johnyaku** [comment]
Yes, you are correct: the samples used in the paper are those listed in the [meta data](https://github.com/Swarbricklab-code/BrCa_cell_atlas/blob/main/spatial_analysis/Meta_Data.csv), namely:
- 182964_1142243F
- 182968_1160920F
- CID4290
- CID4465
- CID44971
- CID4535

I have removed the stereoscope results for the unused samples to avoid confusion. Thanks for bringing this to our attention.

And yes, the default spot indices used by Stereoscope are confusing. There is an option to preserve the barcode spot indices. Due to the ongoing interest in the results from this paper we are reworking this to include the Stereoscope processing, etc. We will look to simplify this in a later release.

**Comment by elliewith** [comment]
Where can I find the deconvolution results for: 
CID4290
CID4465
CID44971
CID4535

I can see the results for:
182964_1142243F
182968_1160920F
